neptune-core/src/application/json_rpc/core/api/ops.rs

```diff
@@ -53,6 +53,27 @@ pub enum RpcMethods {
 
     #[namespace(Namespace::Chain)]
     TipAnnouncements,
+
+    #[namespace(Namespace::Archival)]
+    GetBlock,
+
+    #[namespace(Namespace::Archival)]
+    GetBlockProof,
+
+    #[namespace(Namespace::Archival)]
+    GetBlockKernel,
+
+    #[namespace(Namespace::Archival)]
+    GetBlockHeader,
+
+    #[namespace(Namespace::Archival)]
+    GetBlockBody,
+
+    #[namespace(Namespace::Archival)]
+    GetBlockTransactionKernel,
+
+    #[namespace(Namespace::Archival)]
+    GetBlockAnnouncements,
```

Comment:
Should we drop the "Get" prefix from these names? So `Block` instead of `GetBlock`?

Reply:
It's actually intentional, On Ethereum and similar JSON-RPC APIs, methods without the get_ prefix (like chain_height or node_network) return simple state values that reflect the node’s current state(Like getters on JavaScript), rather than fetching a separate data object

Reply:
Maybe I should document about this naming convention on somewhere?